src/include/daos/rpc.h

```diff
@@ -120,20 +120,10 @@ enum daos_rpc_type {
 };
 
 struct daos_req_comm_in {
-	/** Request user ID, reserved for NRS */
-	uint32_t	req_in_uid;
-	/** Request group ID, reserved for NRS */
-	uint32_t	req_in_gid;
```

Comment:
why remove these fields? plan to use the req_in_paddings in future?

Reply:
Yes.